src/index.html

```diff
@@ -4,9 +4,340 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/styles.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo-container">
+        <a href="#">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <div class="nav-container">
+        <nav class="nav">
+          <ul>
+            <li>
+              <a class="is-active" href="#">Apple</a>
+            </li>
+            <li>
+              <a href="#">Samsung</a>
+            </li>
```

Comment:
```suggestion
            <li class="nav__item">
              <a class="nav__link is-active" href="#">Apple</a>
            </li>

            <li>
              <a href="#">Samsung</a>
            </li>
```

add blank line between multiple sibling blocks and add classes to each list item and link

Reply:
isn't fixed for similar cases